blinkt.go

```diff
@@ -40,6 +47,25 @@ func writeByte(val int) {
 	}
 }
 
+func (bl *Blinkt) InitStopper() {
```

Comment:
Python library calls this something like StopOnExit etc.. can we try to match the name more closely? Also add a comment like:

// InitStopper etc etc etc